### PR TITLE
ansible 2.0.0

### PR DESCRIPTION
**Pull request by trumant**

### CODE REVIEW

**Comment by dunn** [comment]
Unless there's a new 2.1.x development version, you can go ahead and remove the entire `devel do` block. Thanks!

File: Library/Formula/ansible.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ansible < Formula
   desc "Automate deployment, configuration, and upgrading"
   homepage "https://www.ansible.com/home"
-  url "https://releases.ansible.com/ansible/ansible-1.9.4.tar.gz"
-  sha256 "972c2face49f1577bd0ff7989440bfe2820e66fb10d7579915cc536bccfa6fe3"
+  url "https://releases.ansible.com/ansible/ansible-2.0.0.0.tar.gz"
+  sha256 "63a9386ae96bd6ae3e87b9e9fecef3f3f919f333ed8db22292e37df7643cb3f6"
   revision 1
```
Comment (DomT4) [inline Library/Formula/ansible.rb]: `revision` lines can be removed on stable version updates, Thanks!

**Comment by blake** [comment]
Any plan to update the dependent resources such as pywinrm?

**Comment by dunn** [comment]
If they need updating, this would be a good time to do it!

**Comment by trumant** [comment]
I don't see any indications that dependency updates are required. That said, its quite possible that I'm missing how to determine any differences.

**Comment by acaire** [comment]
> I don't see any indications that dependency updates are required. That said, its quite possible that I'm missing how to determine any differences.

pywinrm>=0.1.1 is now recommend as per http://docs.ansible.com/ansible/intro_windows.html#installing-on-the-control-machine mostly because of https://github.com/diyan/pywinrm/pull/74

**Comment by dunn** [comment]
If that's to fix a Windows issue then it's not critical, but feel free to bump the dependencies if you'd like.

**Comment by dunn** [comment]
Merged in 627ec6ee04b3785828917532e1b105c3e8a1b0a2, but feel free to follow up with a update to the dependencies.  Thank you for the contribution!

**Pull request closed without merging**